app/src/main/java/com/example/qstreak/ui/OnboardingSignupFragment.kt

```diff
@@ -29,24 +29,17 @@ class OnboardingSignupFragment : Fragment() {
             null,
             false
         )
-        binding.continueButton.setOnClickListener {
-            createUser()
+        binding.lifecycleOwner = activity
+        binding.viewModel = onboardingViewModel
+        binding.signUpCallback = {
```

Comment:
Instead of manually setting the click listener I prefer to send the method reference to the layout

Reply:
Makes sense, that would be more consistent